tests/utils.py

```diff
@@ -66,6 +66,16 @@ def testing_data_config(*keys):
     return reduce(operator.getitem, keys, _test_data_config)
 
 
+def get_testing_algo_template_path():
+    """
+    a local folder to the testing algorithm template or a url to the compressed template file.
+    Default to None, which effectively uses bundle_gen's ``default_algo_zip`` path.
+
+    https://github.com/Project-MONAI/MONAI/blob/1.1.0/monai/apps/auto3dseg/bundle_gen.py#L380-L381
+    """
+    return os.environ.get("MONAI_TESTING_ALGO_TEMPLATE", None)
```

Comment:
I missed the notification and did not get back in time. Apologies.

I wonder what the plan is for the "local folder". Is it going to be git-cloned from the latest research-contribution repo?

Reply:
@wyli

Reply:
sure, this has the flexibility of using a git-cloned folder as well as a url for the released zip, the goal is to test the latest algo template with monai dev or milestones, do you think it's feasible/meaningful?

Reply:
Thanks @wyli .

I think it is meaningful, especially via the git-cloned folder way, because one limitation today is that I seem to be the only person to zip the templates in research-contribution repo and post it [here](https://github.com/Project-MONAI/research-contributions/releases/tag/algo_templates). The algorithm templates only get covered by tests after this manual step, which I think should be optimized.